INTERNATIONALIZATION.md

```diff
@@ -0,0 +1,121 @@
+# OpenShift Internationalization
+
+#### i18next and react-i18next
+
+Internationalization is implemented with [i18next](https://www.i18next.com/) and [react-i18next](https://react.i18next.com/).
+
+The react-i18next library offers a [hook](https://react.i18next.com/latest/usetranslation-hook), [higher order component](https://react.i18next.com/latest/withtranslation-hoc), and [function](https://react.i18next.com/latest/trans-component) that can be used with React components.
+
+With these tools, we can translate text without having to worry about a key naming strategy. The text is used as the key.
+
+Internationalized text and translations are located in JSON files in a locales folder in each package, as well as a locales folder in the public folder.
+
+In general, these files shouldn't be manually updated, though sometimes plurals will need to be adjusted manually
+after files are generated.
+
+To generate and update text files:
+```
+cd frontend
```

Comment:
Hm, I don't see a `frontend` folder in this repo. Accidentally copied from openshift/console?

Reply:
Thanks, removing